tests/test__version__.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from datetime import datetime
+
+from screenpy_playwright import __version__
+
+
+def test_metadata() -> None:
+    assert __version__.__title__ == "screenpy_playwright"
+    assert __version__.__license__ == "MIT"
+    assert __version__.__author__ == "Perry Goy"
+
+
+def test_copyright_year() -> None:
+    current = datetime.now().year
+    assert f"{current}" in __version__.__copyright__
```

Comment:
Easy test!

... however, can we add a blank line between the "arrange" and "assert" sections? :P :P